src/journey-maps/angular/journey-maps.ts

```diff
@@ -799,9 +803,10 @@ export class SbbJourneyMaps implements OnInit, AfterViewInit, OnDestroy, OnChang
       .pipe(debounceTime(200), takeUntil(this._destroyed))
       .subscribe(() => this.zoomLevelsChange.emit(this._getZooomLevels()));
 
-    this._mapCenterChangeDebouncer
-      .pipe(debounceTime(200), takeUntil(this._destroyed))
-      .subscribe(() => this.mapCenterChange.emit(this._map.getCenter()));
+    this._mapMovementDebouncer.pipe(debounceTime(200), takeUntil(this._destroyed)).subscribe(() => {
+      this.mapCenterChange.emit(this._map.getCenter());
+      this.mapBoundsChange.emit(this._map.getBounds());
```

Comment:
`getBounds()` returns a [`LngLatBounds`](https://maplibre.org/maplibre-gl-js-docs/api/geography/#lnglatbounds) object. I would suggest to emit `getBounds().toArray()`.

It's easier for our clients (especially the ones using the web component) to handle a plain array.

Reply:
We should also refactor `mapCenterChange` accordingly. (But not now as it is a breaking change)